composer.json

```diff
@@ -7,48 +7,50 @@
     "prefer-stable": true,
     "require": {
         "php": ">=7.3",
-        "drupal/core": "^8.9.14",
-        "drupal/linkit": "~5.0-beta8",
-        "drupal/maxlength": "~1.0@beta",
+        "drupal/core": "^8.9.14 || ^9.1",
+        "drupal/linkit": "~5.0-beta12",
+        "drupal/maxlength": "~1.0-rc1",
         "easyrdf/easyrdf": "1.0.0 as 0.9.2",
         "openeuropa/oe_media": "~1.12",
-        "openeuropa/rdf_skos": "~1.0"
+        "openeuropa/rdf_skos": "1.x-dev"
     },
     "require-dev": {
         "composer/installers": "~1.5",
-        "drupal/core-composer-scaffold": "^8.8",
+        "drupal/core-composer-scaffold": "^8.9 || ^9",
         "drupal/address": "~1.7",
         "drupal/composite_reference": "~1.0-alpha2",
         "drupal/config_devel": "~1.2",
-        "drupal/datetime_testing": "1.x-dev",
-        "drupal/description_list_field": "1.0.x-dev",
+        "drupal/datetime_testing": "dev-3217303-drupal-9-compatibility",
+        "openeuropa/description_list_field": "1.0.x-dev",
         "drupal/drupal-extension": "~4.0",
         "drupal/entity_browser": "^2.5",
         "drupal/entity_embed": "~1.0",
         "drupal/entity_reference_revisions": "~1.3",
         "drupal/field_group": "~3.0",
         "drupal/inline_entity_form": "~1.0-rc9",
-        "drupal/typed_link": "~1.1",
-        "drupaltest/behat-traits": "dev-8.x-1.x",
-        "drush/drush": "~9.0",
+        "drupal/typed_link": "1.x-dev",
+        "drupaltest/behat-traits": "dev-GH-6",
+        "drush/drush": "~10.3",
         "friends-of-behat/service-container-extension": "^1.0",
-        "guzzlehttp/guzzle": "~6.3",
         "instaclick/php-webdriver": "^1.4.7",
         "nikic/php-parser": "^3.1.5",
         "openeuropa/behat-transformation-context": "~0.1",
-        "openeuropa/code-review": "~1.5.0",
-        "openeuropa/drupal-core-require-dev": "^8.9.14",
-        "openeuropa/oe_corporate_countries": "~2.0",
-        "openeuropa/oe_corporate_site_info": "~1.0",
-        "openeuropa/oe_link_lists": "~0.5.0",
-        "openeuropa/oe_multilingual": "~1.5",
-        "openeuropa/oe_time_caching": "^1.0.0",
+        "openeuropa/code-review": "~1.6.0",
+        "openeuropa/drupal-core-require-dev": "^8.9.14 || ^9",
+        "openeuropa/oe_corporate_countries": "dev-EWPP-914",
+        "openeuropa/oe_corporate_site_info": "dev-EWPP-1137",
+        "openeuropa/oe_link_lists": "master",
+        "openeuropa/oe_multilingual": "master",
+        "openeuropa/oe_time_caching": "master",
```

Comment:
These components already include the Drupal 9 compatibility fixes in their latest release.

Reply:
"openeuropa/oe_time_caching" has to be used from master branch since new version isn't released yet.